pubspec.yaml

```diff
@@ -1,14 +1,15 @@
 name: webdriver
-version: 0.8.8
+version: 0.9.0
 author: Dart Team <misc@dartlang.org>
 description: >
   Provides WebDriver bindings for Dart. These use the WebDriver JSON interface,
   and as such, require the use of the WebDriver remote server.
 homepage: https://github.com/google/webdriver.dart
 environment:
-  sdk: '>=1.0.0 <2.0.0'
+  sdk: '>=1.9.0-dev.4.0 <2.0.0'
```

Comment:
Can you publish the current version (`0.8.8`), so that pre-1.9 SDKs can play around with this library, before 1.9 hits stable?